client-js/app/elements/labrad-plot.html

```diff
@@ -26,9 +27,9 @@
   <template>
     <div id="plot"></div>
     <div id="buttons">
-      <paper-icon-button icon="tab-unselected" id="rect" on-click="rectMode" title="drag to zoom"></paper-icon-button>
-      <paper-icon-button icon="open-with" id="pan" on-click="panMode" title="drag to pan"></paper-icon-button>
-      <paper-icon-button icon="home" id="home" on-click="reset" title="zoom out"></paper-icon-button>
+      <paper-icon-button icon="tab-unselected" id="rect" on-click="mouseModeZoomRect" title="drag to zoom"></paper-icon-button>
```

Comment:
UI nit: eventually it would be nice to have the [cursor change](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor) to reflect the selected operation when over the corresponding plot. Something like "crosshair" for drag zoom and the "grab" for panning.